### PR TITLE
Ing psd2 adapter

### PR DESCRIPTION
**Pull request by artemsnisarenko**

### CODE REVIEW

File: adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/IngMapper.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import java.util.UUID;
+
+@Mapper
+public interface IngMapper {
+    AccountList map(AccountsResponse value);
+
+    ReadAccountBalanceResponse map(BalancesResponse value);
+
+    @Mapping(target = "balances", ignore = true)
+    @Mapping(target = "links", ignore = true)
```
Comment (n0rdy) [inline adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/IngMapper.java]: I'm not 100% sure, but it seems to me, that one of the `@Mapping` will be ignored without `@Mappings` wrapping, as we use 1.2.0 MapStruct version (https://mapstruct.org/documentation/1.2/reference/html/#basic-mappings), not the 1.3.0 (https://mapstruct.org/documentation/dev/reference/html/#basic-mappings)

File: adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/Headers.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        this.headers = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+        this.headers.putAll(headers);
+    }
+
+    public String getAccessToken() {
+        String authorization = headers.get("Authorization");
+        if (authorization == null || !authorization.startsWith("Bearer ")) {
+            throw new IllegalArgumentException(authorization);
+        }
+        return authorization.substring(7);
```
Comment (n0rdy) [inline adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/Headers.java]: Could you extract this `7` into some variable with a meaningful name or leave a comment above the `authorization.substring(7);` piece of code, please? As this statement is not so clear for future generations. Thanks!

File: adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/IngMapper.java (hunk 1 above)
Comment (artemsnisarenko) [reply adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/IngMapper.java]: From the first link:
> When using Java 8 or later, you can omit the @Mappings wrapper annotation and directly specify several @Mapping annotations on one method.

File: adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/Headers.java (hunk 2 above)
Comment (artemsnisarenko) [reply adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/Headers.java]: Please see the latest changes.

Comment (artemsnisarenko) [reply adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/Headers.java]: https://github.com/adorsys/xs2a-adapter/compare/772a65914155f59fb525faa6ef37d04dd33cafa3..18242af31e8f52c59d2411f3be409b676bd42161

File: adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/IngMapper.java (hunk 1 above)
Comment (n0rdy) [reply adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/IngMapper.java]: Got it, thanks!

**Pull request merged**